src/components/earnings/RedeemHoloFuelCard.vue

```diff
@@ -39,11 +58,34 @@ interface StepTwoProps {
   partialRedemptionTermsAccepted: boolean
 }
 
-function handleSubmit(): void {
+async function handleSubmit(): Promise<void> {
   if (step.value === 1) {
     step.value = 2
   } else {
-    // TODO: submit form
+    isBusy.value = true
+
+    await dashboardStore.getEarnings()
+
+    if (Number(redeemableHoloFuel.value) < Number(amount.value)) {
+      partialRedemptionTermsAccepted.value = false
+      step.value = 1
+      isBusy.value = false
+      return
+    }
+
+    const transaction: RedemptionTransaction = await redeemHoloFuel({
+      amount: amount.value,
+      note: JSON.stringify({ eth_address: hotAddress })
```

Comment:
```suggestion
      wallet_address: hotAddress
```

The endpoint implementation has now been updated such that we no longer need to send the `eth_address` in the `note` field.  Instead you'll need to pass it in as `wallet_address` as shown above.